mne/time_frequency/tfr.py

```diff
@@ -904,6 +904,10 @@ def tfr_multitaper(inst, freqs, n_cycles, time_bandwidth=4.0,
 class _BaseTFR(ContainsMixin, UpdateChannelsMixin, SizeMixin, TimeMixin):
     """Base TFR class."""
 
+    def __init__(self):
+        super().__init__()
```

Comment:
none of the mixins have `__init__` methods (right?), so why is it necessary to call `super().__init__()` here?

Reply:
Good point, it isn't.